ui-react/src/components/topBar/TopBar.tsx

```diff
@@ -1,16 +1,50 @@
-import React from 'react';
-import NavBar from 'react-bootstrap/Navbar';
-import { Container } from 'react-bootstrap';
-import logo from '../../assets/logoFishOnly.svg';
+import React, { useState } from "react";
+import NavBar from "react-bootstrap/Navbar";
+import { Container, Stack } from "react-bootstrap";
+import logo from "../../assets/logoFishOnly.svg";
+import { openExternal } from "../../eventsBridge";
+import { repository } from "../../../package.json";
+import { discordchannel } from "../../../package.json";
+import NavButton from "./NavButton";
+import styles from "./TopBar.module.css";
+import ReportProblemModal from "./ReportProblemModal";
+import Image from "react-bootstrap/Image";
 
 function TopBar() {
+  const [show, setShow] = useState(false);
 
   return (
-    <NavBar bg={'light'}>
-      <Container>
-        <NavBar.Brand><img src={logo} alt="כספיון" width={50} height={50} /></NavBar.Brand>
-      </Container>
-    </NavBar>
+    <>
+      <NavBar
+        className={styles.topNavBar}
+        style={{ borderBottom: "1px solid #C4C4C4" }}
+      >
+        <Container fluid>
+          <NavBar.Brand>
+            <Image
```

Comment:
The logo should probably be a div with background instead of an image. For the same reasons as the button + icon